topic-configuration/README.adoc

```diff
@@ -0,0 +1,689 @@
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
+
+[id="proc-editing-topic-properties_{context}"]
+== Reviewing and editing topic properties in {product}
+
+[role="_abstract"]
+Use the {product} web console to select a topic in your Kafka instance and review the topic properties. You can adjust the editable topic properties as needed.
+
+As an alternative to using the {product} web console, you can use the `rhoas` command-line interface (CLI) to update certain topic properties, as shown in the following example command:
+
+.Example CLI command to update topic retention time
+[source]
+----
+rhoas kafka topic update my-kafka-topic --retention-ms 704800000
+----
+
+For a list of topic properties that you can update using the CLI, see the `rhoas kafka topic update` entry in the {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^].
+
+.Prerequisites
+* You are logged in to the {product} web console.
+* You have created a Kafka instance with at least one Kafka topic in {product}.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the topics that you want to configure.
+. In the *Topics* page, click the name of the topic that you want to review or edit.
++
+Alternatively, select the options icon (three vertical dots) for the relevant topic and click *Edit*.
+. Click *Edit properties*, review the current topic properties, and adjust any editable topic properties as needed.
+. Click *Save* to finish.
+
+[role="_additional-resources"]
+.Additional resources
+* {base-url}{getting-started-url}[_Getting started with {product}_^]
+* {base-url}{rhoas-cli-url}[_Getting started with the `rhoas` CLI_^]
+* {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^]
+
+
+[id="ref-supported-topic-properties_{context}"]
+== Supported topic properties in {product}
+
+[role="_abstract"]
+The following topic properties are supported in {product}. Each listed topic property indicates whether the property is editable or read only, and includes other relevant property attributes for your reference.
+
+=== Core configuration
+
+These properties determine the identity and core behavior of the topic. Before deploying your topic, consider entering all core configuration details for optimal results.
+
+Name::
++
+--
+The topic name is the unique identifier for the topic within the cluster. You need this to set up your producers and consumers, so make it something memorable.
+
+.Property attributes
+[cols="25%,75%"]
+|===
+h|Editable
+|Yes
+
+h|Type
+|String
+
+h|Default value
+|None
+
+h|Supported values
+|Letters (Aa-Zz), numbers, underscores ( _ ), or hyphens ( - ), maximum of 249 characters
```

Comment:
```suggestion
|Letters (Aa-Zz), numbers (0-9), underscores ( _ ), or hyphens ( - ), maximum of 249 characters
```
so initially I thought numbers were not included because I just saw the characters within the parentheses, so even if obvious maybe adding the parentheses also for numbers will make it easy to read it